### PR TITLE
Rename param to align with Go param

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: helm/dns-operator-aws/values.yaml
Comment (calvix) [inline helm/dns-operator-aws/values.yaml:19]: dont we need to change some config repo as well? and maybe mc-bootstrap?

Comment (calvix) [reply helm/dns-operator-aws/values.yaml:19]: https://github.com/giantswarm/config/blob/main/installations/thunder/apps/dns-operator-aws/configmap-values.yaml.patch

Comment (calvix) [reply helm/dns-operator-aws/values.yaml:19]: https://github.com/giantswarm/mc-bootstrap/blob/update-proxy-stack/scripts/setup-config-branch.sh#L242

Comment (fiunchinho) [reply helm/dns-operator-aws/values.yaml:19]: What's missing so we can merge mc-bootstrap changes into main? It's hard to keep track of what's on every branch on different repositories.

Comment (fiunchinho) [reply helm/dns-operator-aws/values.yaml:19]: Param change on mc-bootstrap https://github.com/giantswarm/mc-bootstrap/pull/366

Comment (calvix) [reply helm/dns-operator-aws/values.yaml:19]: I think the main reason we have this stale branch is that we wanted stable mc-bootstrap in order to be able to create talanx  MCs, as rocket is doing lot of changes for the CI and there was an agreement to create a branch that we know works for us.

Comment (fiunchinho) [reply helm/dns-operator-aws/values.yaml:19]: Param change on the config repo https://github.com/giantswarm/config/pull/1465
We can't merge this PR until this one is merged and released.

Comment (calvix) [reply helm/dns-operator-aws/values.yaml:19]: `thunder` ahs anyway stopped all flux and updates so no merge of PR will update anything there

**Review by calvix — APPROVED** [review]

**Pull request merged**
